tests/integration/build_test.go

```diff
@@ -0,0 +1,30 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/rancher/rio/tests/testutil"
+)
+
+func buildTests(t *testing.T, when spec.G, it spec.S) {
+
+	var service testutil.TestService
+
+	when("an image is built locally in rio", func() {
+		it.Before(func() {
+			service.CreateLocal(t)
```

Comment:
`CreateLocal` is somewhat ambiguous when reading it here. How about `BuildAndCreate` or `CreateFromDockerfile`